src/components/Activities/Activities.tsx

```diff
@@ -0,0 +1,73 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+
+export default function Activities() {
```

Comment:
Please rename the file, the folder, and the component to `ActivitiesList`

Sorry I forgot to note this in the ticket!